File: AlphaAppv0.1/src/tn/esprit/models/Commande.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package tn.esprit.models;

import java.util.Date;

/**
 *
 * @author asus
 */
public class Commande {
    private int IdCommande;
    private int IdClient;
    private Date DateCom;
    private float MontantCom;
    private String EtatCom;
    
    public Commande() {
    }

    public Commande(int IdCommande) {
        this.IdCommande = IdCommande;
    }
    
    public Commande(int IdClient, Date DateCom, float MontantCom, String EtatCom) {
        this.IdClient = IdClient;
        this.DateCom = DateCom;
        this.MontantCom = MontantCom;
        this.EtatCom = EtatCom;
    }

    public Commande(Date DateCom, float MontantCom, String EtatCom) {
        this.DateCom = DateCom;
        this.MontantCom = MontantCom;
        this.EtatCom = EtatCom;
    }
    
    
    public Commande(int IdCommande, int IdClient, Date DateCom, float MontantCom, String EtatCom) {
        this.IdCommande = IdCommande;
        this.IdClient = IdClient;
        this.DateCom = DateCom;
        this.MontantCom = MontantCom;
        this.EtatCom = EtatCom;
    }

    public int getIdCommande() {
        return IdCommande;
    }
    
    public void setIdCommande(int IdCommande) {
        this.IdCommande = IdCommande;
    }

    public int getIdClient() {
        return IdClient;
    }

    public void setIdClient(int IdClient) {
        this.IdClient = IdClient;
    }

    public Date getDateCom() {
        return DateCom;
    }

    public void setDateCom(Date DateCom) {
        this.DateCom = DateCom;
    }

    public float getMontantCom() {
        return MontantCom;
    }

    public void setMontantCom(float MontantCom) {
        this.MontantCom = MontantCom;
    }

    public String getEtatCom() {
        return EtatCom;
    }

    public void setEtatCom(String EtatCom) {
        this.EtatCom = EtatCom;
    }
    @Override
    public String toString(){
        return "Commandes[" + "ID: " +IdCommande +" ID Client: " + IdClient + " Date commande: "
                + DateCom + " Montant commande: " + MontantCom +" Etat commande: " + EtatCom;}
}

File: src/Entity/Commande.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;

/**
 * Commande
 *
 * @ORM\Table(name="commande", indexes={@ORM\Index(name="fk_idClient", columns={"IdClient"})})
 * @ORM\Entity
 */
class Commande
{
    /**
     * @var int
     *
     * @ORM\Column(name="IdCommande", type="integer", nullable=false)
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="IDENTITY")
     */
    private $idcommande;

    /**
     * @var string
     *
     * @ORM\Column(name="numc", type="string", length=20, nullable=false)
     */
    private $numc;

    /**
     * @var int
     *
     * @ORM\Column(name="IdClient", type="integer", nullable=false)
     */
    private $idclient;

    /**
     * @var \DateTime
     *
     * @ORM\Column(name="DateCom", type="date", nullable=false)
     */
    private $datecom;

    /**
     * @var float
     *
     * @ORM\Column(name="MontantCom", type="float", precision=10, scale=0, nullable=false)
     */
    private $montantcom;

    /**
     * @var string
     *
     * @ORM\Column(name="EtatCom", type="string", length=20, nullable=false)
     */
    private $etatcom;

    public function getIdcommande(): ?int
    {
        return $this->idcommande;
    }

    public function getNumc(): ?string
    {
        return $this->numc;
    }

    public function setNumc(string $numc): self
    {
        $this->numc = $numc;

        return $this;
    }

    public function getIdclient(): ?int
    {
        return $this->idclient;
    }

    public function setIdclient(int $idclient): self
    {
        $this->idclient = $idclient;

        return $this;
    }

    public function getDatecom(): ?\DateTimeInterface
    {
        return $this->datecom;
    }

    public function setDatecom(\DateTimeInterface $datecom): self
    {
        $this->datecom = $datecom;

        return $this;
    }

    public function getMontantcom(): ?float
    {
        return $this->montantcom;
    }

    public function setMontantcom(float $montantcom): self
    {
        $this->montantcom = $montantcom;

        return $this;
    }

    public function getEtatcom(): ?string
    {
        return $this->etatcom;
    }

    public function setEtatcom(string $etatcom): self
    {
        $this->etatcom = $etatcom;

        return $this;
    }


}
